clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/narrowing-conversions.rst

```diff
@@ -27,6 +27,29 @@ This check will flag:
  - All applications of binary operators with a narrowing conversions.
    For example: ``int i; i+= 0.1;``.
 
+Note that arithmetic with integer types may perform implicit conversions if the used integer types are smaller than ``int``.
```

Comment:
Consider something like this (after correcting links):
`
Arithmetic with smaller integer types than ``int`` may trigger implicit conversions, as explained under "Integral Promotion" on the cppreference.com page. The example below demonstrates this behavior.
`

Reply:
Updated.